apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/html/libraries/iframe-edit-mode.css.ts

```diff
@@ -448,6 +448,35 @@ export const getEditPageCss = (timestampId: string): string => {
         border: 1px solid #53c2f9 !important;
         display: block;
     }
+    
+    .navbar-form {
```

Comment:
Is missing the `${timestampId}` selector.

Also change this for something more specific like `.dotcms__navbar-form`.

Reply:
@fmontes done!